cardano-cli/src/Cardano/CLI/Commands/Governance.hs

```diff
@@ -56,11 +58,61 @@ data GovernanceCmdError
   | GovernanceCmdDecoderError !DecoderError
   | GovernanceCmdVerifyPollError !GovernancePollError
   | GovernanceCmdWriteFileError !(FileError ())
-  -- Legacy - remove me after cardano-cli transitions to new era based structure
-  | ShelleyGovernanceCmdMIRCertNotSupportedInConway
-  | ShelleyGovernanceCmdGenesisDelegationNotSupportedInConway
+
+  | GovernanceCmdMIRCertNotSupportedInConway
+  | GovernanceCmdGenesisDelegationNotSupportedInConway
   deriving Show
 
+renderGovernanceError :: GovernanceCmdError -> Text
```

Comment:
That's already implemented as a part of https://github.com/input-output-hk/cardano-cli/pull/84/files